apps/ejabberd/src/mod_mam_muc_odbc_arch.erl

```diff
@@ -151,7 +151,8 @@ archive_message_unsafe(Host, MessID, RoomID, FromNick, Packet) ->
     SMessID = integer_to_list(MessID),
     TextBody = mod_mam_utils:packet_to_search_body(mod_mam_muc, Host, Packet),
     STextBody = mongoose_rdbms:escape(TextBody),
-    write_message(Host, SMessID, SRoomID, SFromNick, SData, STextBody).
+    BTextBody = unicode:characters_to_binary(STextBody),
```

Comment:
I'd rather make sure that escaping always returns an utf8 binary; less special cases everywhere.

Reply:
A very good idea! Felipe, can you please implement it when you're done with ODBC issue?